maps/southern_cross/southern_cross_jobs_vr.dm

```diff
@@ -0,0 +1,24 @@
+var/const/PATHFINDER 		=(1<<13) //VOREStation Edit - Added Pathfinder
+
+/obj/item/weapon/card/id/science/head/pathfinder
+	desc = "A card which represents discovery of the unknown."
+	assignment = "Pathfinder"
+	rank = "Pathfinder"
+	job_access_type = /datum/job/pathfinder
+
+/datum/job/pathfinder
+	title = "Pathfinder"
+	flag = PATHFINDER
+	department = "Science"
+	department_flag = MEDSCI
+	faction = "Station"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "the research director"
+	selection_color = "#AD6BAD"
+	idtype = /obj/item/weapon/card/id/science/head/pathfinder
+	economic_modifier = 10
```

Comment:
10 is a head position value. Even QM only gets 5. They should have something between 5 and 10 probably, but not the same as, for example, the CE which this is. Team leads IRL *might* get paid more than their coworkers, but even then, maybe not. It's more symbolic and manager-in-training than actual manager.